SYCL/ESIMD/api/esimd_bit_ops.cpp

```diff
@@ -0,0 +1,146 @@
+//==------- esimd_bit_ops.cpp  - DPC++ ESIMD on-device test ----------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu
+// UNSUPPORTED: cuda
+// RUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
```

Comment:
```suggestion
// RUN: %clangxx -fsycl %s -o %t.out
```
Is the option mandatory? if not it is better to remove it. To let us run testing with different values of this option.

Reply:
Yes, this option is needed to test all cases even if some lead to BE compilation failure.